### ISSUE TITLE
Missing facts referenced by infra.pp when using Ubuntu 14.04 AWS AMI 

### ISSUE THREAD

**Issue by waoliveros** [claranet/puppet-newrelic#11]
## Affected Puppet, Ruby, OS and module versions/distributions

- Puppet: 4.10.8
- Ruby:
- Distribution: Ubuntu 14.04 AWS AMI
- Module version: 2.0.0

## How to reproduce (e.g Puppet code you use)

I am installing the module using 
`include newrelic` 
with hiera params for 
```newrelic::license_key
newrelic::enable_php_agent
newrelic::agent::php::ini_settings:
  appname
  daemon.loglevel
newrelic::php_conf_dir
newrelic::php_purge_files
```

It seems like infra.pp:32 looks into `$facts['os']['lsb']['distcodename']`.
Running `facter`, we only get the following for `os`

```
os => {
  architecture => "amd64",
  distro => {
    codename => "trusty",
    description => "Ubuntu 14.04.5 LTS",
    id => "Ubuntu",
    release => {
      full => "14.04",
      major => "14.04"
    }
  },
  family => "Debian",
  hardware => "x86_64",
  name => "Ubuntu",
  release => {
    full => "14.04",
    major => "14.04"
  },
  selinux => {
    enabled => false
  }
}
```

## What are you seeing

Puppet run encounters an error.

## What behaviour did you expect instead

NewRelic should be installed.

## Output log

`Error: Could not retrieve catalog from remote server: Error 500 on SERVER: Server Error: Evaluation Error: Operator '[]' is not applicable to an Undef Value. at /etc/puppetlabs/code/environments/development/modules/newrelic/manifests/repo/infra.pp:32:22`

**Comment by craigwatson** [comment]
Hi,

Thanks for raising this - I've just fixed the bug and released version 2.0.1 to the Forge :)

Craig

**Comment by waoliveros** [comment]
Awesome. Thanks, Craig!
